source/env_tutorial/bsuite.rst

```diff
@@ -0,0 +1,190 @@
+Bsuite
+~~~~~~~
+
+Description
+=======
+
+``bsuite`` is a collection of carefully-designed experiments that investigate core capabilities of a reinforcement learning (RL) agent with two main objectives:
+
+    -  1. To collect clear, informative and scalable problems that capture key issues in the design of efficient and general learning algorithms.
+    -  2. To study agent behavior through their performance on these shared benchmarks.
+
+.. image:: ./images/bsuite.png
+   :align: center
+
+Installation
+====
+
+How to install
+--------
+
+You just need to use the command ``pip`` to install bsuite, however it will be automatically installed when installing DI-engine.
+
+.. code:: shell
+
+   # Method1: Install Directly
+   pip install bsuite
+   # Method2: Install with DI-engine requirements
+   cd DI-engine
+   pip install ".[common_env]"
+Verify installation
+--------
+
+Once installed, you can verify whether the installation is successful by running the following command on the Python command line.
+
+.. code:: python
+
+   import bsuite
+   env = bsuite.load_from_id('catch/0')
+   timestep = env.reset()
+   print(timestep)
+Original environment space
+========================
+
+.. _Observations space-1:
+Observations space
+--------
+
+-  Array representing the state of the environment, dimensions and size can vary according to the specific environment, datatype is \ ``np.float32``.
+
+.. _Actions space-1:
+Actions space
+--------
+
+-  The action space is a discrete space of size N, (where N can vary according to the environment), datatype is \ ``int``\ and input is a python integer value（or a np array of dimension 0 such as \ ``np.array(1)``\ to input action 1）.
+
+-  For example, in the Deep Sea environment, N is equal to 2, thus action values ranges from 0 to 1. For their specific meaning you can refer to the following list.
+
+   -  0：LEFT.
+
+   -  1：RIGHT.
+
+.. _Rewards space-1:
+Rewards space
+--------
+
+-  Rewards are assigned according to the rules of the environments, rewards are usually a \ ``float``\ value.
+
+.. _Other-1:
+Others
+----
+
+-  Environments terminate once they have reached their maximum number of steps or is encountered a failure state. All environments have a fixed number of maximum steps, but not all environments have a failure state.
+
+Key facts
+========
+
+1. Each environment contains several configurations to make it gradually more challenging.
+
+2. Discrete actions space.
+
+3. Each environment is designed to test a particular propriety of RL policies, including: generalization, exploration, credit assignment, scaling, noise, memory.
+
+4. Rewards scale can vary a lot.
```

Comment:
Replace 'a lot' by 'significantly' since 'a lot' is a little bit informal and oral

Reply:
The scale of rewards might be better